README.md

```diff
@@ -1,7 +1,205 @@
 # recidiviz
 An in-progress effort to visualize recidivism metrics.
 
-## Data
+## Contents
+1. [Development](#development)
+1. [Tooling](#tooling)
+1. [Application Structure](#application-structure)
+1. [Data Sources](#data-sources)
+1. [License](#license)
+
+## Development
+
+### Getting set up
+1. Grab the source:
+
+   ```git clone git@github.com:Recidiviz/recidiviz.com.git```
+1. Install Yarn package manager:
+
+   ```brew install yarn```
+
+   For alternative Yarn installation options, see [Yarn Installation](https://yarnpkg.com/en/docs/install).
+1. Install dependencies:
+
+   ```yarn install```
+
+That's it! We suggest installing a linting package for your preferred code editor that hooks into [eslint](#eslint). We recommend [linter-eslint](https://atom.io/packages/linter-eslint) for Atom.
+
+### Linting & running tests
+A yarn script is available to run both [eslint](#eslint) and tests via [Jest](#jest):
+
+```yarn test```
+
+This is the generally advisable way to roll. This task will also be executed prior to any `git commit` or `git push` action using [husky](#husky). Running tests this way will also write code coverage statistics to stdout and the `coverage` directory.
+
+To run eslint manually:
+
+```eslint src```
+
+To run Jest manually:
+
+```jest [optional-filepath]```
+
+### Running the application locally
+A yarn script is available for starting the development server. This will also automatically open a browser to localhost on the appropriate port.
+
+```yarn start```
+
+Each time this is run, the `build` directory will be wiped clean. A [bundle analysis](#Bundle-analysis) report, found in `build/report.html` will also be generated on each invocation of this script.
```

Comment:
Need a comma after `build/report.html`, and in the equivalent sentence about 10 lines down.